### ISSUE TITLE
Wake Up Word

### ISSUE THREAD

**Issue by PhDinBlunts** [athena-voice/athena-voice-client#51]
I would like to change the wake up word to "help". I followed the instructions as best I could, but it does not work for me. I changed the WAKE_UP_WORD in settings.py to "help", I uncommented the POCKET_DICT =      join(DATA_DIR, 'athena.dict') line, and I editted the athena.dict so the only line in it is "help HH EH L P"

After all that, it still doesnt work. Any advice? Below are pictures of the editted code.

![athena dict](https://cloud.githubusercontent.com/assets/25036333/21814001/f0807c00-d725-11e6-824d-b241582f6900.png)
![settings 1](https://cloud.githubusercontent.com/assets/25036333/21814005/f225a670-d725-11e6-80cd-a7edaa7341df.png)
![settings 2](https://cloud.githubusercontent.com/assets/25036333/21814008/f36e50ae-d725-11e6-91d2-578150573238.png)

**Comment by rcbyron** [comment]
It looks like there is an extra space before the second "POCKET_DICT" line which should throw a syntax error in Python. Other than that, everything looks good. I just tried it on my machine and it works fine with "help HH EH L P" added. Did the word "athena" work? Does it hang on the "Waiting to be woken up..." part? If so, a better microphone might help as the key word recognition is far from perfect. I've noticed it helps to say the wake up word at different pitches/speeds/loudness.
